toolchain/sem_ir/builtin_function_kind.h

```diff
@@ -0,0 +1,50 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef CARBON_TOOLCHAIN_SEM_IR_BUILTIN_FUNCTION_KIND_H_
+#define CARBON_TOOLCHAIN_SEM_IR_BUILTIN_FUNCTION_KIND_H_
+
+#include <cstdint>
+
+#include "common/enum_base.h"
+
+namespace Carbon::SemIR {
+
+class File;
+struct InstId;
```

Comment:
Can some of these be included instead of forward declared?

Reply:
Done.